metricbeat/module/system/raid/blockinfo/getdev.go

```diff
@@ -19,14 +19,13 @@ package blockinfo
 
 import (
 	"fmt"
-	"io/ioutil"
 	"os"
 	"path/filepath"
 )
 
 // ListAll lists all the multi-disk devices in a RAID array
 func ListAll(path string) ([]MDDevice, error) {
```

Comment:
```suggestion
func ListAll(logger *logp.Logger, path string) ([]MDDevice, error) {
```

Reply:
Resolving since I'll go with the partial metrics error solution.